web/server/api/endpoints/lineage.py

```diff
@@ -54,17 +51,17 @@ async def column_lineage(
     context: Context = Depends(get_loaded_context),
 ) -> t.Dict[str, t.Dict[str, LineageColumn]]:
     """Get a column's lineage"""
-    model_fqn = normalize_model_name(model_name, context.default_catalog, context.config.dialect)
+    model_name = normalize_model_name(model_name, context.default_catalog, context.config.dialect)
```

Comment:
this should just be

model = context.get_model(model_name)

then you can get the fqn with model.fqn